source/extensions/filters/http/compressor/BUILD

```diff
@@ -27,6 +27,7 @@ envoy_cc_extension(
     name = "config",
     srcs = ["config.cc"],
     hdrs = ["config.h"],
+    extra_visibility = ["//test/extensions:__subpackages__"],
```

Comment:
was this still necessary, and if so can we narrow it as much as possible?

Reply:
will remove. i think i missed this one during previous cleanup. thanks for catching